tests/conftest.py

```diff
@@ -224,7 +224,7 @@ async def guard_func(*args: _P.args, **kwargs: _P.kwargs) -> _R:
 
 # Guard a few functions that would make network connections
 location.async_detect_location_info = check_real(location.async_detect_location_info)
-util.get_local_ip = lambda: "127.0.0.1"
+util.get_local_ip = lambda: "127.0.0.1"  # type: ignore[attr-defined]
```

Comment:
This I purposefully left out. I think it is fully unused and can be completely removed from `conftest.py`
I think it is patching a function that has been removed...

Reply:
I'll leave it out this PR then